docs/tutorials/06-batch-infer.ipynb

```diff
@@ -624,38 +616,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "tsdb_profile = DatastoreProfileV3io(name=\"v3io-tsdb-profile\")\n",
-    "project.register_datastore_profile(tsdb_profile)\n",
+    "from src.model_monitoring_utils import enable_model_monitoring\n",
     "\n",
-    "stream_profile = DatastoreProfileV3io(\n",
-    "    name=\"v3io-stream-profile\",\n",
-    "    v3io_access_key=mlrun.mlconf.get_v3io_access_key(),\n",
-    ")\n",
-    "project.register_datastore_profile(stream_profile)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "a8c47b9d",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "project.set_model_monitoring_credentials(\n",
-    "    tsdb_profile_name=tsdb_profile.name,\n",
-    "    stream_profile_name=stream_profile.name,\n",
-    ")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "b14395bb-be2b-49f7-acf3-85387dbe29d9",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "project.enable_model_monitoring(\n",
-    "    base_period=1, wait_for_deployment=True, deploy_histogram_data_drift_app=False\n",
+    "# Disabling old model monitoring to update configurations\n",
```

Comment:
If model monitoring was enabled in v1.7.x: disable model monitoring to update the configurations